rust_src/src/remacs_sys.rs

```diff
@@ -105,6 +105,8 @@ extern "C" {
     pub fn allocate_misc(t: Lisp_Misc_Type) -> LispObject;
     #[cfg(windows)]
     pub fn file_attributes_c(filename: LispObject, id_format: LispObject) -> LispObject;
+    #[cfg(windows)]
```

Comment:
Did you have to add this for it to compile?

Reply:
I didn't try without it as it looked like other W32 functions were there as well so I followed the pattern. I can try removing it but as neither I nor CI build on Windows, I don't know if we'll actually notice if it doesn't work.

Reply:
Just realised you probably just mean the `cfg` bit. It compiles without it, so it should be fine, I've added another commit to do that.

Reply:
Yeah, we used to have a working Windows CI but it was too flaky. I want to get one working again. Just not sure where to host it. Most people want to build things with Windows tools and we want to build with Rust and friends. Which is what broke the previous CI.